README.md

```diff
@@ -296,6 +296,14 @@ remote_branch: refs/publish/master/msync_foo
 pre_commit_script: openstack-commit-msg-hook.sh
 ```
 
+###### AWS Codecommit
+
+```
+---
+gitbase: https://git-commit.us-east-1.amazonaws.com/
+branch: modulesyncbranch
+```
+
```

Comment:
This worked for me when I was testing with CodeCommit recently:

```yaml
---
git_base: 'https://git-codecommit.us-west-2.amazonaws.com/'
namespace: v1/repos
branch: modulesync
message: "Update from my-modulesync-github"
remote: https://git-codecommit.us-west-2.amazonaws.com/v1/repos/my-iac-github

# vim: syntax=yaml```

Reply:
That works I guess, although it feels a bit un-DRY

Reply:
@joestump the `remote` bit won't work if you want to manage more than one repo